### PR TITLE
Add json output to action.yml

### LINKED ISSUE #166: action.yml has not json object declared
#### Steps to reproduce this issue

1. Use a workflow with using the json
2. run actionlint
3. See a warning, that json is not a valid output of the metadata action

#### Expected behaviour

Everything in the output object is defined in the action.yml

#### Actual behaviour

The documentation says that are there is an [json output object](https://github.com/docker/metadata-action#json-output-object), 
but in the [action.yml](https://github.com/docker/metadata-action/blob/master/action.yml) the is not defined

This results into warnings in [actionlint](https://github.com/rhysd/actionlint) if the value is used.

### Configuration

* Repository URL (if public): 
* Build URL (if public): 

```yml
      - name: Docker meta web
        id: meta-web
        uses: docker/metadata-action@v3
        with:
          images: ghcr.io/stefanschoof/espressoweb
          tags: |
            type=schedule
            type=ref,event=branch
            type=ref,event=tag
            type=ref,event=pr
            type=raw,value=latest,enable=${{ endsWith(github.ref, github.event.repository.default_branch) }}
          bake-target: docker-metadata-action-web

      - name: Start container
        run: docker run -d -p 80:80 --health-interval=5s --name web ${{ fromJSON(steps.meta-web.outputs.json).tags[0] }}
```

### Logs

> Download the [log file of your build](https://docs.github.com/en/actions/managing-workflow-runs/using-workflow-run-logs#downloading-logs)
> and [attach it](https://docs.github.com/en/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests) to this issue.

### PR DESCRIPTION
**Pull request by StefanSchoof**
Resolves #166

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -42,6 +42,8 @@ outputs:
     description: 'Generated Docker labels'
   bake-file:
     description: 'Bake definiton file'
+  json:
+    description: 'A JSON object composed of the generated tags and labels'
```
Comment (crazy-max) [inline action.yml:46]: Better to keep the same desc of the README: https://github.com/docker/metadata-action#outputs

```suggestion
    description: 'JSON output of tags and labels'
```

**Review by crazy-max — CHANGES_REQUESTED** [review]

**Comment by StefanSchoof** [comment]
Okay, did this via web from my phone. The DCO seems not to like this. Will try to take a look, when I at my PC. Probably not today

**Comment by crazy-max** [comment]
> Okay, did this via web from my phone. The DCO seems not to like this. Will try to take a look, when I at my PC. Probably not today

Yes and also squash commits, thx.

**Review by crazy-max — APPROVED** [review]
LGTM thanks

**Pull request merged**